Formula/openfpgaloader.rb

```diff
@@ -0,0 +1,26 @@
+class Openfpgaloader < Formula
+  desc "Universal utility for programming FPGA"
+  homepage "https://github.com/trabucayre/openFPGALoader"
+  url "https://github.com/trabucayre/openFPGALoader.git",
+    tag:      "v0.4.0",
+    revision: "1dbbd34420e00070bd0f58e0a91a653e12a87b97"
+  license "Apache-2.0"
+  revision 1
+  head "https://github.com/trabucayre/openFPGALoader.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libftdi"
+  depends_on "libusb"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    version_output = shell_output("#{bin}/openFPGALoader -V 2>&1")
+    assert_match "openFPGALoader v#{version}", version_output
```

Comment:
Would it be possible to add a functional test here, other than comparing version or help output? These rarely fail for broken installs.

Reply:
openFPGALoader relies on external hardware. AFAIK there is no sensible test we could do without having a USB device physically connected to the CI machine...

@trabucayre: Anything to add?

Reply:
As a reference, stlink also relies on the version string as its sole functionality test: https://github.com/Homebrew/homebrew-core/blob/master/Formula/stlink.rb

Reply:
There are quite a few formulae with just version and help tests, and they could do with better tests – unless it really isn't possible to. See https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula.

Reply:
Can we try to access a non-existent USB device and then check we get the expected failure and exit code?

Reply:
Sorry for this late answer.
It's possible to use `openFPGALoader --detect`: with this option it try to access an generic **FT2232** device and without hardware connected fails with a message and a return code equal to `EXIT_FAILURE`

Reply:
Thanks all for your feedback. I added a little test using ` --detect`.